### DISCUSSION TITLE
Grafana - Playlists

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#2121] [Unsupported Versions] [answered]
I tried creating a playlist in Grafana to rotate through all my nodes to display the status. However, when I click on Create, I get a Permission Denied error.

Is there a way to modify the user account to give it the permissions needed to create playlists?

Thanks

Joe

**Comment by dougburks** [comment, marked as answer]
Have you tried logging into Grafana as described at https://docs.securityonion.net/en/2.3/grafana.html#accounts?

**Reply by joeb1kenobe** [reply]
Thanks Doug. Logging in as admin worked.
